web/src/router/index.jsx

```diff
@@ -5,11 +5,17 @@ import Index from "../pages/Index";
 import SignIn from "../pages/auth/SignIn";
 import SignUp from "../pages/auth/SignUp";
 import ResetPassword from "../pages/auth/ResetPassword";
+import ErrorPage from "../pages/error/ErrorPage";
 
 export default function Router() {
   return (
     <Routes>
+      {/* error page */}
+      <Route path="*" element={<ErrorPage />} />
```

Comment:
Huhhh, you'll have to revert it to the previous state, '*' path is used to catch pages that don't exist, 404